lua/starfall/libs_sv/constraint.lua

```diff
@@ -145,14 +156,22 @@ function constraint_library.axis(e1, e2, bone1, bone2, v1, v2, force_lim, torque
 	end
 end
 
---- Ballsocket two entities
+--- Ballsocket two entities together. For more options, see constraint.ballsocketadv
+-- @param e1 The first entity
+-- @param e2 The second entity
+-- @param bone1 Number bone of the first entity
+-- @param bone2 Number bone of the second entity
+-- @param pos Position of the joint, relative to the second entity
```

Comment:
Need to verify this is local to second entity

Reply:
It is indeed relative to the second entity